fetchmail_attach_from_folder/model/fetchmail_server.py

```diff
@@ -75,111 +68,109 @@ def fetch_mail(self, cr, uid, ids, context=None):
             connection = this.connect()
             for folder in this.folder_ids:
                 this.handle_folder(connection, folder)
-
             connection.close()
 
         return super(fetchmail_server, self).fetch_mail(
             cr, uid, check_original, context)
 
-    def handle_folder(self, cr, uid, ids, connection, folder, context=None):
+    @api.multi
+    def handle_folder(self, connection, folder):
         '''Return ids of objects matched'''
 
         matched_object_ids = []
 
-        for this in self.browse(cr, uid, ids, context=context):
-            logger.info('start checking for emails in %s server %s',
-                        folder.path, this.name)
+        for this in self:
+            logging.info(
```

Comment:
throughout this module you are using the `logging` module shortcut methods, which by default use the root logger. This is unusual in Odoo addons and can make log filtering configuration more difficult (AFAIR if I change the logging level of the root logger, this has an impact on all its descendents).

I think you should use a named logger. If you don't want to use the `_logger = logging.getLogger(__name__)` idiom, fine, but you still should not use the root logger.